src/main/native/org_bblfsh_client_v2_libuast_Libuast.cc

```diff
@@ -15,6 +15,8 @@ jobject asJvmBuffer(uast::Buffer buf) {
   return env->NewDirectByteBuffer(buf.ptr, buf.size);
 }
 
+const char *nativeContext = "nativeContext";
```

Comment:
Minor nit: Either `const char * const nativeContext = "…"` or `const char nativeContext[] = "…"` is preferable here—to ensure the pointer itself is also const (either works). `constexpr char nativeContext[] = "…"` will also work if we assume C++ ≥ 11.

Reply:
Yes, thank you, we do! Was thinking about `constexpr` as well - will replace as part of the next PR.